lib/index.js

```diff
@@ -19,6 +19,20 @@ var render = require('posthtml-render')
  */
 function PostHTML (plugins) {
   this.plugins = plugins || []
+
+  /**
+   * @example
```

Comment:
Yep sure, but external modules/ plugins in general should not be part of the core docs when possible :)

```js
/**
 * PostHTML Instance
 * 
 * @name posthtml
 * @version 0.9.1
 * 
 * @prop plugins
 * @prop options
 */
this.name = 'posthtml'
this.version = '0.9.1'

this.plugins = plugins || []
this.options = {}
```